src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/RazorLanguageServer.cs

```diff
@@ -55,34 +54,47 @@ private RazorLanguageServer(ILanguageServer innerServer)
 
         public Task WaitForExit => _innerServer.WaitForExit;
 
-        public Task InitializedAsync(CancellationToken token) => _innerServer.InitializedAsync(token);
+        public Task InitializedAsync(CancellationToken token) => _innerServer.Initialize(token);
```

Comment:
Does this give a compiler warning due to `Async` in title on a non-`async` function? => Worth surrounding in pragmas?

Reply:
No warnings, I assume those are based off of Task rather than async.